npm/private/npm_import.bzl

```diff
@@ -272,16 +272,11 @@ def npm_link_imported_package_internal(
         dev,
         root_package,
         link,
-        link_packages,
         public_visibility,
         npm_link_imported_package_store_macro,
         npm_imported_package_store_macro):
     bazel_package = native.package_name()
 
```

Comment:
This is the (unwrapped) method that is documented as `npm_link_imported_package` from each npm package. For example in the root BUILD we have:
```
load("@acorn__8.4.0__links//:defs.bzl", npm_link_acorn = "npm_link_imported_package")
```

which we pass to:

```
npm_link_all_packages(
    imported_links = [
        npm_link_acorn,
    ],
)
```

However I don't think the `link_packages` param deleted from this method is ever used, so that might not even be a breaking change (where removing `npm_import(link_packages)` is breaking).